crates/sui-graphql-rpc/src/client/simple_client.rs

```diff
@@ -48,7 +59,7 @@ async fn test_client() {
             chainIdentifier
         }
     "#;
-    let res = client.execute(query.to_string()).await.unwrap();
+    let res = client.execute(query.to_string(), vec![]).await.unwrap();
```

Comment:
you may want to provide an api for optionally providing headers, otherwise everywhere you don't want to provide headers you're still forced to create an empty vec (which is cheap but is more an annoyance)